### DISCUSSION TITLE
How do we know if we passed task 1?

### DISCUSSION THREAD

**Discussion by ElliottMiles** [SimDynamX/SpaceTeamsPRO#67] [2024 Lunar SAR Competition Q&A] [answered]
How do we know if we passed task 1 and if we need to re-submit?

**Comment by connorjak** [comment, marked as answer]
Task 1 submission reviews are still in progress; please continue on to Task 2.

**Comment by connorjak** [comment]
If there's something wrong with your Task 1 or Task 2 submission, we'll let you know so you can fix it for the (scored) Final task submission. No need to resubmit.
